src/plugins.js

```diff
@@ -9,12 +9,31 @@ import requestAuthPlugin from './plugins/requestAuthPlugin';
  * @module ServerPlugins
  */
 
+export function setCsrfCookies(request, reply) {
+	const csrfHeader = (request.response.headers || {})['x-csrf-jwt'];
+	if (csrfHeader) {
+		reply.state('x-csrf-jwt-header', csrfHeader);
+	}
+	return reply.continue();
+}
+
 export function getCsrfPlugin(secret) {
 	const register = (server, options, next) => {
-		server.state('x-csrf-jwt', {
+		const cookieOptions = {
+			path: '/',
 			isSecure: process.env.NODE_ENV === 'production',
-		});
-		return CsrfPlugin.register(server, options, next);
+		};
+		server.state('x-csrf-jwt', {
+			...cookieOptions,
+			isHttpOnly: true,
+		});  // set by plugin
+		server.state('x-csrf-jwt-header', {
```

Comment:
`server.state` just sets the default cookie options - the cookies are actually created by `reply.state()` calls for a particular request (i.e. [global cookie config](https://hapijs.com/tutorials/cookies?lang=en_US#configuring-the-server) vs. [setting the cookie](https://hapijs.com/tutorials/cookies?lang=en_US#setting-a-cookie))

- `electrode-csrf-jwt` server plugin calls `reply.state` to set `'x-csrf-jwt'`
- the `onPreResponse` handler will call `reply.state` to set `'x-csrf-jwt-header'`

These two `server.state` configuration calls just make sure that the 'cookie options' are in sync for both cookies.

I'll add docs